tests/Etc/TestSeeder.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Stancl\Tenancy\Tests\Etc;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class TestSeeder extends Seeder
```

Comment:
Did Shift remove this file completely? Or why is it added as a new file in this PR?

Reply:
> Did Shift remove this file completely?

No. Shift did the same with this file as with others. 

> Or why is it added as a new file in this PR?

Keeping the `TestSeeder` class in the tests file threw the error `Unable to resolve Database/Seeder/SeederTest class.`  
Laravel was trying to resolve the file from Database/Seeder directory, maybe because pest tests are not namespaced. So I extracted the class into a separate file (namespaced), and it worked fine.

Reply:
Ah so you just moved it? It shows up as a new file in the diff.

Reply:
> Ah so you just moved it?

Yes.